rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1252,6 +1258,193 @@ def __init__(
 
         self._prepare_layers()
 
+    def adjust_sparse_layers(
+        self,
+        data_example: Dict[Text, List[FeatureArray]],
+        new_feature_sizes: Dict[Text, Dict[Text, List[int]]],
+        old_feature_sizes: Dict[Text, Dict[Text, List[int]]],
```

Comment:
Would be helpful to give an example schema for the `new_feature_sizes` / `old_feature_sizes` dictionaries in the docstring, just like you did in the last PR.